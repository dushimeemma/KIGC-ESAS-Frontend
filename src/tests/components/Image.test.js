import React from 'react';
import { shallow } from 'enzyme';

import HomeImage from '../../components/Image';

describe('<HomeImage/>', () => {
  it('Should render navigation', () => {
    const wrapper = shallow(<HomeImage />);
    expect(wrapper).toMatchSnapshot();
  });
});
