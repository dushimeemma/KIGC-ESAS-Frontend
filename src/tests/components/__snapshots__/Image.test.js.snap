// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeImage/> Should render navigation 1`] = `
<img
  alt="HomePage"
  className="col-md-6"
  src=""
/>
`;
